helm-chart/dash0-operator/tests/operator/deployment-and-webhook_test.yaml

```diff
@@ -242,3 +274,43 @@ tests:
           value: ghcr.io/dash0hq/configuration-reloader@sha256:3e8c25853217c7393dbd95e17fe2117bb31b39478bbea4479cc5e7c1257dda03
       - notExists:
           path: spec.template.spec.containers[0].env[6].name
+
+  - it: webhook have caBundle set
+    documentSelector:
+      path: kind
+      value: MutatingWebhookConfiguration
+    asserts:
+      - isNotNullOrEmpty:
+          path: webhooks[0].clientConfig.caBundle
+
+  - it: webhook have caBundle set also with custom settings
+    documentSelector:
+      path: kind
+      value: MutatingWebhookConfiguration
+    set:
+      operator:
+        webhookPort: 554
+    asserts:
+      - hasDocuments:
+          count: 5
+      - isNotNullOrEmpty:
+          path: webhooks[0].clientConfig.caBundle
+
+  - it: injector service should render the "dash0.cert-digest" label
+    documentIndex: 4
+    asserts:
+      - isNotNullOrEmpty:
+          path: spec.selector["dash0.cert-digest"]
+      - matchSnapshot: {}
```

Comment:
q: Is it intentional that this test has a "normal" assertion `isNotNullOrEmpty` as well as a `matchSnapshot` assertion? I think it does not really make sense. The matchSnapshot checks all attributes anyway, so the isNotNullOrEmpty is redundant. Or if this test is only meant to check for the selector, maybe remove the matchSnapshot.

Reply:
matchSnapshot can go